operations/companies.md

```diff
@@ -111,9 +123,18 @@ Returns all company profiles of the enterprise, possibly filtered by identifiers
 | `TaxIdentifier` | string | optional | Tax identification number of the company. |
 | `AdditionalTaxIdentifier` | string | optional | Additional tax identifier of the company. |
 | `ElectronicInvoiceIdentifier` | string | optional | Electronic invoice identifier of the company. |
+| `InvoiceDueInterval` | string | optional | The maximum time, when the invoice has to be be paid in ISO 8601 duration format. |
 | `AccountingCode` | string | optional | Accounting code of the company. |
-| `Address` | [Address](configuration.md#address) | optional | Address of the company \(if it is non-empty, otherwise `null`\). |
+| `MotherCompanyId` | guid | optional | Unique identifier of mother company. |
+| `CreatedUtc` | string | optional | Date of [Company](#company) creation date and time. |
+| `UpdatedUtc` | string | optional | Date of [Company](#company) last update date and time. |
 | `BillingCode` | string | optional | Billing code of the company. |
+| `Iata` | string | optional | Iata of the company. |
+| `Telephone` | string | optional | Contact telephone number. |
+| `InvoicingEmail` | string | optional | Email for issuing invoices to the company. |
+| `ContactPerson` | string | optional | Contact person of the company. |
+| `Contact` | string | optional | Contact of the company. |
```

Comment:
What is `contact`? I assume it's not the name of a person, because we have a separate field called `ContactPerson`. And I don't think it's an email address, because these fields normally have the word 'email' in them. So... what is it? A fuller description is needed, I think.

Reply:
Hi @fpindej, are you able to answer this question about `Contact`? Thank you.

Reply:
@MikeAdamsMews this is freeform field usually used for a telephone or email address

Reply:
Based on that information, I've updated the description to "Other contact details, such as telephone, email or similar." in the three places it occurs.